vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java

```diff
@@ -1626,14 +1633,42 @@ public Grid(Class<T> beanType) {
      *            the data communicator builder type
      * @param <U>
      *            the GridArrayUpdater type
+     * @deprecated since 24.9. In Vaadin 25, this constructor will continue to
+     *             exist without the {@code updateQueueBuilder} parameter.
```

Comment:
Mention to the new constructor missing:

```suggestion
     * @deprecated since 24.9 and will be removed in Vaadin 25. Use
     *             {@link #Grid(Class, DataCommunicatorBuilder)}
     *  
```

Reply:
Thanks, fixed.